packages/worker/src/logger.ts

```diff
@@ -43,7 +45,8 @@ function defaultLogFunction({ level, timestampNanos, message, meta }: LogEntry):
 /**
  * Takes a `[seconds, nanos]` tuple as returned from getTimeOfDay and turns it into bigint.
  */
-export function timeOfDayToBigint([seconds, nanos]: [number, number]): bigint {
+export function timeOfDayToBigint(timeOfDay: [number, number]): bigint {
```

Comment:
I don't think we expose this to users, we can keep it they way it was

Reply:
I only updated functions that show up in search:

https://typescript.temporal.io/search?q=__namedPara

![image](https://user-images.githubusercontent.com/251288/143922407-78ceb87c-f6f8-4434-80e3-426f2a6fcb14.png)